src/java/htsjdk/variant/variantcontext/GenotypeLikelihoods.java

```diff
@@ -72,6 +72,13 @@
     */
     private final static GenotypeLikelihoodsAllelePair[] PLIndexToAlleleIndex = calculatePLcache(MAX_ALT_ALLELES_THAT_CAN_BE_GENOTYPED);
 
+
+    /*
+    * a cache of the PL index to the anyploid alleles it represents over all possible numbers of alternate alleles
+    */
+    protected final static List<List<Integer>> plIndexToAlleleIndices = new ArrayList<List<Integer>>();
```

Comment:
Yes, the ploidy would have to be saved. My logic is to only deal with one ploidy. With the `map`, the application developer could initialize for multiple ploidys, which would waste memory.